README.md

```diff
@@ -1,5 +1,37 @@
 # Helm Charts for Scalar DL
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) ![Release Charts](https://github.com/scalar-labs/helm-charts/workflows/Release%20Helm%20Charts/badge.svg?branch=main)
+
 This directory contain helm charts for scalardl.
 
 * [Scalar DL](./charts/scalardl/)
+
+
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+Once Helm is set up properly, add the repo as follows:
+
+```console
+helm repo add scalardl https://scalar-labs.github.io/helm-charts
```

Comment:
You will need to change the repo name as well. https://github.com/scalar-labs/scalar-kubernetes/commit/34f25c5a2688aee3bf3a78c79b43f5fc92ed99ed